DevelopGitHub/GitHubExtension/GitHubExtension.Infrastructure.ActionFilters/Extensions/ActivityFiltersExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Web.Http.Dependencies;
+using System.Web.Http.Filters;
+using GitHubExtension.Infrastructure.ActionFilters.Constants;
+using GitHubExtension.Infrastructure.ActionFilters.Models;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Infrastructure.ActionFilters.Extensions
+{
+    public static class ActivityFiltersExtensions
+    {
+        public static IDependencyResolver GetDependencyResolver(this HttpActionExecutedContext actionExecutedContext)
+        {
+            return actionExecutedContext.ActionContext.RequestContext.Configuration.DependencyResolver;
```

Comment:
Can you reso;ve all dependencies through the SimpleInjector intead of spreading resolving instances in different places? You you should have only one point of injection and resolving.